### ISSUE TITLE
Stop command looks wrong or misleading

### ISSUE THREAD

**Issue by davygravy** [davygravy/roomba-mqtt-homeassistant#5]
Opcode 173 in the OI

> Stop Opcode: 173 Data Bytes: 0
This command stops the OI. All streams will stop and the robot will no longer respond to commands.
Use this command when you are finished working with the robot.
 Serial sequence: [173].
 Available in modes: Passive, Safe, or Full
 Changes mode to: Off. Roomba plays a song to acknowledge it is exiting the OI. 

This is referenced in 
```

void clean() {
  debugV("Sending 'clean' command");
  Serial.write(128);
  delay(50);
  Serial.write(135);
}

void cleanMax() {
  debugV("Sending 'cleanMax' command");
  Serial.write(128);
  delay(50);
  Serial.write(136);
}

void cleanSpot() {
  debugV("Sending 'cleanSpot' command");
  Serial.write(128);
  delay(50);
  Serial.write(134);
}

void seekDock() {
  debugV("Sending 'seekDock' command");
  Serial.write(128);
  delay(50);
  Serial.write(143);
}

void stop() {
  debugV("Sending 'stop' command");
  Serial.write(128);
  delay(50);
  Serial.write(173);
}

void powerOff() {
  debugV("Sending 'powerOff' command");
  Serial.write(128);
  delay(50);
  Serial.write(133);
}
```
and called by ;

```
void onMQTTMessage(char* topic, byte* payload, unsigned int length) 
{
  String newTopic = topic;
  payload[length] = '\0';
  String command = String((char *)payload);
  debugV("MQTT received command: %s", command.c_str());
  if (newTopic == MQTT_COMMAND_TOPIC) {
    wakeUp();

    // Official Home Assistant commands
    if (command == "start" || command == "pause" || command == "start_pause" || command == "stop") {
      clean();
    }
    else if (command == "clean_spot") {
      cleanSpot();
    }
    else if (command == "return_to_base") {
      seekDock();
    }
    else if (command == "turn_off") {
      powerOff();
    }
  }
}
```
So the `powerOff()` command using the wrong opcode.  It probably should be `133` :

> Power Opcode: 133 Data Bytes: 0
This command powers down Roomba. The OI can be in Passive, Safe, or Full mode to accept this
command.
 Serial sequence: [133]
 Available in modes: Passive, Safe, or Full
 Changes mode to: Passive

Also need to check that what is being set as payload text is correct/factual.

**Comment by davygravy** [comment]
Likely fixed in [commit 2cc2176a534e4973e1c15dda76d21ee058e578d7](https://github.com/davygravy/roomba-mqtt-homeassistant/commit/2cc2176a534e4973e1c15dda76d21ee058e578d7).  

Leaving it open in case:
- the `stop()` function can be removed
- there was any other naming inconsistency related to the stop/pause/start/power_down confusing between HA commands and robot commands.

**Comment by davygravy** [comment]
Fixed and tested.  Makes a big difference in the usability of the HA interface.  The STOP button now actually stops the device dead in its tracks, rather than just starting a new cleaning run.
